src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -257,9 +259,18 @@ public void autonomousInit() {
     public void teleopInit() {
         if (SubsystemConstants.DRIVE_ENABLED) {
             subsystems.drivebaseSubsystem.setDefaultCommand(new DriveCommand(subsystems.drivebaseSubsystem,
-                    controls.driveController.getLeftXAxis(), controls.driveController.getLeftYAxis(),
+                    controls.driveController.getLeftYAxis(), controls.driveController.getLeftXAxis(),
                     controls.driveController.getRightXAxis()));
         }
+        if (!isCompetition())
+            return;
+        if (INTAKE_ENABLED) {
```

Comment:
Check whether intakeSubsystem is non-null vs. using another variable to infer if intakeSubsystem is valid or not.